src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -601,6 +634,22 @@ where
         Ok([new_folded_X0, new_folded_X1])
     }
 
+    fn fold_step_circuit_instances(
```

Comment:
NBD: strictly speaking, what we defined here is no longer folding operation but a customized version of accumulation. but this is just minor thing